srcpkgs/elogind/patches/SEEK_DATA.patch

```diff
@@ -0,0 +1,23 @@
+Upstream:no
+
+Our musl version doesn't expose the SEEK_DATA and SEEK_HOLE in `unisted.h`...
+Only version >= 1.2.3 do that.
+
+--
+diff --git a/src/shared/copy.h b/src/shared/copy.h
+index a34a87c..60c63f8 100644
+--- a/src/shared/copy.h
++++ b/src/shared/copy.h
+@@ -70,6 +70,12 @@ static inline int copy_bytes(int fdf, int fdt, uint64_t max_bytes, CopyFlags cop
+         return copy_bytes_full(fdf, fdt, max_bytes, copy_flags, NULL, NULL, NULL, NULL);
+ }
+
++#ifndef __GLIBC__
```

Comment:
It seems better to do `#ifndef SEEK_DATA` and `#ifndef SEEK_HOLE` here.

Reply:
at the time, because its musl only, made sense to use `__GLIBC__`  but you are absolutely right ! 

Updated the PR